packages/smartgpt-bridge/src/log-stream.ts

```
@@ -3,32 +3,39 @@ import path from "node:path";
-  if (outputs.length === 1) return outputs[0]!;
+	if (outputs.length === 1) return outputs[0]!;
 
-  return new Writable({
-    write(chunk, _enc, cb) {
-      for (const out of outputs) out.write(chunk);
-      cb();
-    },
-  });
+	return new Writable({
+		write(chunk, _enc, cb) {
+			let pending = outputs.length;
```

ESLint

[error] Unexpected let, use const instead.

(functional/no-let)